best-practices-trove-project-creation.md

```diff
@@ -0,0 +1,64 @@
+
+**Project Name** - keep this brief but concise ideally containing the object(s) and environment desired for example "Cigarettes on Beaches," "Clothing Stores" or "Pleasant Memories - Food."
+
+**Description** - provide a description in prose form highlighting the key objects in the desired context and environments to be captured in the photos you are looking for in as much detail as possible.  Also call out what the types of photos will _NOT_ be accepted.  Please note our requirement to indicate if the photos you desire will contain face data.  For example - "Send photos of a SPAM can in multiple places in your house (kitchen, living room, bedroom, bathroom). In each place, take at least 5 photos from different sides of the can (front, back, top, left, and right).  Do not send photos of a SPAM can on a shelf in a grocery store or photos containing people." 
+
+**How Photos will be used** - similar to the description, provide a summary in prose form of how these photos will be used and for what purpose.  For example - "These photos will be used to train artificial intelligence (AI) models to help improve the recognition of broken glass mugs and vessels in images for safety alerts."
+
+
+
+## Photo Requirements
+List concise and clear "Do's" and "Don'ts" of the photos you are asking for.  
+
+**EXAMPLE**
+
+Do’s
+- Submit photos taken with your phone camera only
+- Capture the 5 sides of the object, but it is okay if you are not looking straight on to each side
+- Only take images of a whole SPAM can
+- Feel free to move the item or the phone to get the different angles
+- Be creative with where you put item in the listed environments, i.e., on your kitchen table, kitchen counter, on the living room sofa, etc.
+- Vary your distance to the camera, but don’t get too far where the can is super small or too close where the can takes up most of the image
+
+Don’ts
+- Don’t send blurry, far away, or very dark photos
+- Don’t send photos found on Google, Flickr, Bing, or stock photos, etc. (these will be rejected)
+- Don’t send photos with identifiable people (i.e., no faces), but it is okay if you hold the item and only your hand is seen
+- Don’t send photos with personally identifiable information (names, phones, etc.)
+
+## Example Photos
+
+The sample photos you upload should be highly representative of the data you need in terms of quality and content.  Make sure the photos are clearly labeled as **GOOD** or **BAD** samples and where possible provide a brief caption explaining the reason why such as "GOOD: Shows doors closed, no people in photo" or "BAD: Door open, people in photo"
+
+![](/Buyer-Sample-Good-1.png)![](/Buyer-Sample-Bad-1.png)
+
+## Setting the Price per Photo
+
+Most projects pay between USD$0.50 to USD$5.00 per photo depending on the availability and complexity of the requirements.  You should review the examples of similar successful projects to help set the price per photo of your project.
```

Comment:
Buyers can be outside of the US. We might say that most pay between USD 0.5 and USD 5.0 (or equivalent currency).

Reply:
Will do!

Reply:
Done.